release_notes/370-release-notes.adoc

```diff
@@ -158,6 +160,8 @@ In the table, features are marked with the following statuses:
 * When configuring a JFrog Artifactory integration, the username and password fields are now optional to allow anonymous pulls. (**ROX-10090**)
 * Validation to the web user interface for endpoint URLs in the generic webhook integration caused errors. This issue was fixed. (**ROX-9902**)
 * The policy *OpenShift: Kubeadmin Secret Accessed* is no longer triggered if the request was from the default OpenShift `oauth-apiserver-sa` service account, because this is an expected access pattern for the OpenShift API server. (**ROX-10018**)
+* The ability to enable notifications for multiple policies selected in the **Policies** list has been reinstated. To enable notifications, select one or more policies and choose Enable notification from the Bulk Actions menu. (**ROX-9985**)
+* Read/write permission issue for vulnerability reports has been fixed. (**ROX-9880**)
```

Comment:
Has this been left vague intentionally? If some permission requirement has been dropped now, I believe it is worthwhile specifying the permissions that are no longer required to use reports.

Reply:
It looks like the issue is a bug where users with read/write permissions could not use some features in vulnerability reports. I've rewritten it to be less vague and posted to the slack channel to get feedback.

Reply:
Rewritten with input from Van.